Replace functions and classes deprecated in Octave 7 and removed from Octave 9.

diff -uNr a/src/Makefile b/src/Makefile
--- a/src/Makefile	2021-07-05 01:05:54.000000000 -0400
+++ b/src/Makefile	2021-08-20 00:58:10.329056319 -0400
@@ -15,7 +15,7 @@
 
 AR := $(shell $(MKOCTFILE) -p AR)
 
-PKG_CXXFLAGS := -Wall -Wno-deprecated-declarations $(PKG_CXXFLAGS_APPEND)
+PKG_CXXFLAGS := -Wall $(PKG_CXXFLAGS_APPEND)
 
 all: __control_slicot_functions__.oct \
      __control_helper_functions__.oct
